src/test/testMocking.ts

```diff
@@ -61,6 +64,7 @@ describe('Mock', () => {
       returnListOfListOfIntArg(l: Int): [[Int]]
       returnListOfListOfObject: [[Bird!]]!
       returnStringArgument(s: String): String
+      node(id:String!):Flying
```

Comment:
Yes, but couldn't we support both things? Like if your mock returns a `__typename` that is used (which supports your case) but if you don't want to care about that it will select a random one?